packages/shared/src/components/auth/PostAuthBanner.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactElement } from 'react';
+import { feature } from '../../lib/featureManagement';
+import { checkIsBrowser, checkIsExtension, UserAgent } from '../../lib/func';
+import { AuthExtensionBanner } from './AuthExtensionBanner';
+import { AuthenticationBanner } from './AuthenticationBanner';
+import { useConditionalFeature } from '../../hooks';
+
+export const PostAuthBanner = (): ReactElement => {
```

Comment:
Not sure what but this now introduces a change in prod value already:
[existing-change.webm](https://github.com/user-attachments/assets/d48cdab3-f132-4449-877c-d0b0b89de5ab)

Reply:
@rebelchris I'm not entirely sure I follow ? 😅 You are swapping between prod and preview in that video. Both look as they are supposed to, or ?

If you're referring to the fact that the design changes, we clarified with the design team that the design changes were general changes, and only the CTA is the actual experiment:

[reference](https://dailydotdev.slack.com/archives/C0381BQUEN9/p1732103856077919?thread_ts=1731933888.885839&cid=C0381BQUEN9)

Reply:
Ah thanks, that kind of stuff is good to note, else others might keep saying you introduced a change that's unintentional :)

Reply:
Fair, I'll make sure to note these kind of things in the initial PR opening!